Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2018"
 
 [dependencies]
 bitvec = "0.22"
```

Comment:
unfortunately we can't do the bitvec upgrade everywhere yet because of 
- https://github.com/bitvecto-rs/bitvec/issues/169
- https://github.com/zkcrypto/ff/pull/73

Reply:
Ok, thanks

Reply:
Now that `zcashd 4.7.0` is done, I'm working on this (it requires a breaking `ff` upgrade, which needs to be coordinated with a bunch of crates, including in RustCrypto).

Reply:
@cryptonemo looks like the issue in bitvec is now resolved